.github/workflows/test-build-windows.yml

```diff
@@ -39,8 +39,11 @@ jobs:
         set -e
         set -u
         cd elasticsearch/linux_distributions
-        chmod 755 opendistro-windows-build.sh
-        sh ./opendistro-windows-build.sh
+        ./opendistro-windows-build.sh
+        ls -lrt target/
+        aws s3 cp target/*.exe s3://artifacts.opendistroforelasticsearch.amazon.com/downloads/odfe-windows/staging/odfe-executable/
+        aws s3 cp target/*.zip s3://artifacts.opendistroforelasticsearch.amazon.com/downloads/odfe-windows/staging/odfe-window-zip/
+        aws cloudfront create-invalidation --distribution-id E1VG5HMIWI4SA2 --paths "/downloads/*"
```

Comment:
how does the old logic work if these are newly added?

Reply:
As for the time being I move the logic from the script "`elasticsearch/linux_distributions/opendistro-tar-build.sh`" for consistency

More improvements later.

Reply:
This was a part of opendistro-windows-build.sh file before
https://github.com/opendistro-for-elasticsearch/opendistro-build/blob/master/elasticsearch/linux_distributions/opendistro-windows-build.sh#L56-L58